feature/appdetail/src/main/kotlin/com/merxury/blocker/feature/appdetail/AppDetailScreen.kt

```diff
@@ -205,6 +205,7 @@ fun AppDetailRoute(
                 }
             }
         },
+        onRefresh = viewModel::refresh,
```

Comment:
For reloading the list, if the `load` function is simple enough, you can use the load function directly.